service/controller/app/v1/resource/chart/error.go

```diff
@@ -2,13 +2,13 @@ package chart
 
 import "github.com/giantswarm/microerror"
 
-var failedExecution = &microerror.Error{
+var failedExecutionError = &microerror.Error{
```

Comment:
We call this `executionFailedError` usually.

Reply:
Damn missed that. Its renamed now.